src/main/java/com/accantosystems/stratoss/vnfmdriver/driver/GrantDriver.java

```diff
@@ -123,29 +137,36 @@ public Grant getGrant(String grantId) throws GrantRejectedException, GrantProvid
         headers.setAccept(Arrays.asList(MediaType.APPLICATION_JSON));
         final HttpEntity<Void> requestEntity = new HttpEntity<>(headers);
         final ResponseEntity<Grant> responseEntity;
+        String driverRequestId = "";
+        UUID uuid = UUID.randomUUID();
         try {
             responseEntity = authenticatedRestTemplate.exchange(url, HttpMethod.GET, requestEntity, Grant.class, grantId);
+            if(responseEntity != null)
+                driverRequestId = responseEntity.getBody().getVnfLcmOpOccId();
+            LoggingUtils.logEnabledMDC(grantId, MessageType.REQUEST, MessageDirection.RECEIVED, uuid.toString(),MediaType.APPLICATION_JSON.toString(), "https",getRequestProtocolMetaData(url) ,driverRequestId);
```

Comment:
We are making external call here. why REQUEST is RECEIVED and RESPONSE is Sent in these lines?